### ISSUE TITLE
Open and Close

### ISSUE THREAD

**Issue by amirshahmiri** [brian-the-dev/python-tradingview-ta#4]
Hi, 

Is there a way to get the daily change value? either in percentage or numerical.

Thanks in advance!

**Comment by deathlyface** [comment]
Hello amirshahmiri,

I believe you can't do that. Here is an example of what data you can get using python-tradingview-ta: [TradingView](https://www.tradingview.com/symbols/NASDAQ-NDAQ/technicals/).

Please reply if you have another question, I'm happy to help.

**Comment by deathlyface** [comment]
You can get open and close price directly from an exchange or external API.

**Comment by RaymondTerry** [comment]
I currently am working on a personal project with this project being a central point and also would find it convenient to get the daily percentage change and found this to be a way of achieving it. To be put shortly, you just do it manually. Here is what I am working with. 

`
def percent_change_24h():

    handler = TA_Handler(
        symbol=BTC,
        screener="crypto",
        exchange="COINBASE",
        interval=Interval.INTERVAL_1_DAY
    )

    analysis = handler.get_analysis()

    price = analysis.indicators['open']

    daily_change = ((price - btc_price()) / btc_price()) * 100

    print(price)
    print(btc_price())
    print(daily_change)

   #Example usage
    if daily_change <= -20.0000:
        print("Decent Dip to DCA")

def btc_price():
    

    btc_price = TA_Handler(
        symbol=BTC,
        screener="crypto",
        exchange="COINBASE",
        interval=Interval.INTERVAL_1_MINUTE
    )


    analysis = btc_price.get_analysis()

    price = analysis.indicators['close']

    return price
`

**Comment by brian-the-dev** [comment]
Hey everyone, this issue has been open for nearly a year. Recently, sina-rostami added the "change" indicator in #59. I will release these changes soon.

**Comment by brian-the-dev** [comment]
Released in v3.2.8. Please update tradingview-ta using `pip install -U tradingview-ta`.

To get the daily change value in percentage, set the interval to 1 day. Code example:

```python
from tradingview_ta import TA_Handler, Interval

btcusdt = TA_Handler(
  symbol="BTCUSDT",
  exchange="binance",
  screener="crypto",
  interval=Interval.INTERVAL_1_DAY
)

analysis = btcusdt.get_analysis()
change = analysis.indicators["change"]
print(f"Over the last 24h, the price of BTC/USDT changes by {change}%")
# Output: Over the last 24h, the price of BTC/USDT changes by -0.15517712%
```

I'll close this issue. If you have any questions, please comment below.
